### PR TITLE
Update root.json hash in docs

### PR DESCRIPTION
**Pull request by stmcginnis**
**Issue number:**

N/A

**Description of changes:**

With the switch to the new TUF repo root, this updates all references in the docs and examples to use the new hash.

**Testing done:**

Visual inspection.

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by etungsten — APPROVED** [review]

**Review by bcressey — APPROVED** [review]
Approved, assuming that the new root.json hash is verified after the cache is updated and before this is merged.

**Pull request merged**
